package.json

```diff
@@ -8,31 +8,43 @@
   "license": "MIT",
   "dependencies": {
     "path-root": "^0.1.1",
-    "resolve": "^1.13.1"
+    "resolve": "^1.17.0"
   },
   "devDependencies": {
-    "@types/chai": "^4.2.6",
-    "@types/fs-extra": "^8.0.1",
-    "@types/mocha": "^5.2.7",
-    "@types/node": "^12.12.14",
-    "@types/semver": "^6.2.0",
+    "@types/chai": "^4.2.13",
+    "@types/fs-extra": "^9.0.2",
+    "@types/mocha": "^8.0.3",
+    "@types/node": "^14.11.8",
+    "@typescript-eslint/eslint-plugin": "^4.4.1",
+    "@typescript-eslint/parser": "^4.4.1",
     "chai": "^4.2.0",
-    "execa": "^3.4.0",
-    "fixturify-project": "^1.9.1",
-    "fs-extra": "^8.1.0",
-    "mocha": "^6.2.2",
-    "semver": "^6.3.0",
-    "typescript": "^3.7.3"
+    "eslint": "^7.11.0",
+    "eslint-config-prettier": "^6.12.0",
+    "eslint-plugin-mocha": "^8.0.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "execa": "^4.0.3",
+    "fixturify-project": "^2.1.0",
+    "fs-extra": "^9.0.1",
+    "mocha": "^8.1.3",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^2.1.2",
+    "typescript": "^4.0.3"
   },
   "scripts": {
     "build": "tsc -b .",
     "watch": "tsc --watch .",
     "clean": "tsc -b --clean .",
-    "test": "yarn build && yarn test:js",
+    "test": "npm-run-all build lint test:js",
     "test:js": "mocha tests/*-test.js",
-    "test:js:debug": "mocha debug tests/*-test.js"
+    "test:js:debug": "mocha debug tests/*-test.js",
+    "lint": "eslint ."
   },
   "engines": {
-    "node": "8.* || 10.* || >= 12"
+    "node": "10.* || >= 12"
```

Comment:
Probably should have been:

```json
    "node": "10.* || 12.* || >= 14"
```

Node 13 has been EOL'ed for a while.

Reply:
Also, probably should have been done independently of the linting changes to call out what the breaking change was here...